corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -82,20 +121,25 @@ def handle(self, domain, months, csv, **options):
             .domain(domain).values_list("_id", flat=True)
         )
 
-        self._forms_per_user_per_month()
-        self._cases_created_per_user_per_month()
-        self._cases_updated_per_user_per_month()
-        self._case_to_case_index_ratio()
-        self._ledgers_per_case()
-        self._attachment_sizes()
-        self._ucr()
-
-    def _doc_counts(self):
-        self._print_value('Total cases', CaseES().domain(self.domain).count())
-        self._print_value('Open cases', CaseES().domain(self.domain).is_closed(False).count())
-        self._print_value('Total forms', FormES().domain(self.domain).count())
-
-    def _forms_per_user_per_month(self):
+        self.collect_doc_counts()
+        self.collect_forms_per_user_per_month()
+        self.collect_cases_created_per_user_per_month()
+        self.collect_cases_updated_per_user_per_month()
+        self.collect_case_transactions()
+        self.collect_case_indices()
+        self.collect_synclogs()
+        self.collect_devicelogs()
+        self.collect_ledgers_per_case()
+        self.collect_attachment_sizes()
+        self.collect_ucr_data()
+
+        self.output_stats()
+
+    def collect_doc_counts(self):
+        resource_model['forms_total'] = FormES().domain(self.domain).count()
```

Comment:
Have you reviewed the default filters that are applied to these queries? Some of them might be worth removing

Reply:
Oooh! Good point!